src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_computer.java

```diff
@@ -290,10 +295,10 @@ public boolean checkRecipe_EM(ItemStack itemStack) {
                 mEfficiencyIncrease = 10000;
                 maxCurrentTemp.set(maxTemp);
                 availableData.set(eAvailableData);
-                return true;
+                return SimpleCheckRecipeResult.ofSuccess("no_computing");
```

Comment:
`ofFailure`

Reply:
This one is a bit complicated. Originally the multi returned true, in order to still consume power etc. However it does not generate computation in this case.

Reply:
I see